### ISSUE TITLE
the result is not good

### ISSUE THREAD

**Issue by mapeng1995** [hzm8341/SSD_keras_restnet#1]
hello i  have train your code. It is amazing that you have  draw a structure diagram.
But after I iterated 40 times, I tested some models and found that the results were bad. Basically it is impossible to identify images in training JPGRImages, and the accuracy is quite low. Did you test this model?

**Comment by hzm8341** [comment]
In working ,I didn't use this model.
But I had trained this model ,the result is not so good .
Because the training computer  is not good enough, just  one GPU(1080Ti), I didn't cost much time on this model.

**Comment by mapeng1995** [comment]
Thanks for your reply
there is my test result on voc2007(train) and i set the batch=2 is normal as yours. When the batch is 20,the gpu is out of mermory(Titan 1080ti 11G)   
![img_20180513_153726](https://user-images.githubusercontent.com/34765259/39967858-b8f159b6-56f5-11e8-88a0-80a273acb309.jpg)
How to be that.....i thought the model did not learn anything.   However, the loss function is constantly declining ....
 i think you might be knowing that. Thanks for your help

**Comment by mapeng1995** [comment]
The pictures is the result after training 25 iterations
 
My guess is whether the result of the pre-training weight is not good. I would like to ask what data set training is based on the pre-training weight.

Can you recommend a model that works better? I tried yolov3, SSD, faster-rcnn on my data set, but the effect was general.

**Comment by hzm8341** [comment]
sorry ,I didn't train a good model. I just change the base net from vgg16 to resnet.
I guess the reason mayby is the base network——resnet.
you can change following:
in train_SSD.py file line 232,you will see "model = SSD_resnet.resnet_68(input_shape, num_classes=NUM_CLASSES)",change the "resnet_68" to "resnet_50".

after you training ,if the same question happen, you can try change the code to make new resnet_101.

and also , if you are chinese ,you can add my QQ 175915864,we can chat with chinese.
